### PR TITLE
feat: Tutorials using Pythonic API

### PR DESCRIPTION
**Pull request by aryan-rajoria**
Adding the tutorials using the Pythonic API.

### CODE REVIEW

File: eva/parser/utils.py [hunk 1]
```diff
@@ -46,6 +50,30 @@ def parse_load(table_name: str, file_regex: str, format: str, **kwargs):
...
+        assert isinstance(stmt, DropUDFStatement), "Expected a drop UDF statement"
+    return stmt
+
+
+def parse_create(item_type: str, item_name: str, model_type: str, model: str, **kwargs):
+    if item_type.upper() == "TABLE":
+        assert isinstance(stmt, CreateTableStatement), "Expected a drop table statement"
+    else:
+        mock_query = f"""
+            CREATE {item_type} IF NOT EXISTS {item_name}
```
Comment (gaurav274) [inline eva/parser/utils.py:68]: Make `if_not_exists` an input variable.

File: eva/parser/utils.py [hunk 2]
```diff
@@ -46,6 +50,30 @@ def parse_load(table_name: str, file_regex: str, format: str, **kwargs):
...
+            TYPE  {model_type}
+            'model' '{model}';
+        """
+        stmt = Parser().parse(mock_query)[0]
+        assert isinstance(stmt, CreateUDFStatement), "Expected a drop UDF statement"
```
Comment (gaurav274) [inline eva/parser/utils.py:73]: Incorrect error string.

File: eva/interfaces/relational/db.py [hunk 3]
```diff
@@ -169,6 +199,40 @@ def load(
         stmt = parse_load(table_name, file_regex, format, **kwargs)
         return EVARelation(self._evadb, stmt)
 
+    def drop(self, item_name: str, item_type: str, **kwargs) -> EVARelation:
+        """
+        Drop a table or UDF from the database.
+
+        Args:
+            item_name (str): Name of the table or UDF to be dropped.
```
Comment (gaurav274) [inline eva/interfaces/relational/db.py:207]: I don't like the variable names. It is fine to create 2 APIs, `drop` and `drop_udf` and be precise.

File: eva/interfaces/relational/db.py [hunk 4]
```diff
@@ -169,6 +199,40 @@ def load(
...
+        # DROP {item_type} item_name
+        stmt = parse_drop(item_name, item_type, **kwargs)
+        return EVARelation(stmt)
+
+    def create(self, item_type: str, item_name: str, **kwargs) -> EVARelation:
+        """
+        Create a table or UDF and add it to the database.
+
+        Args:
+            item_type (str): Type of item either: table or udf.
```
Comment (gaurav274) [inline eva/interfaces/relational/db.py:223]: Same as above.

File: eva/interfaces/relational/db.py [hunk 5]
```diff
@@ -169,6 +199,40 @@ def load(
...
+            name (str): Name of the table or UDF to be created.
+            model_type (str): Type of model to be created could be 'classification', 'object-detection', etc.
+            model (str): Path of the model weigths.
+            **kwargs: Additional keyword arguments for configuring the load operation.
+
+        Returns:
+            EVARelation: The EVARelation object representing the loaded table.
+        """
+        # DROP {item_type} item_name
```
Comment (gaurav274) [inline eva/interfaces/relational/db.py:232]: incorrect comment

File: eva/interfaces/relational/db.py [hunk 6]
```diff
@@ -169,6 +199,40 @@ def load(
...
+    def create(self, item_type: str, item_name: str, **kwargs) -> EVARelation:
+        """
+        Create a table or UDF and add it to the database.
+
+        Args:
+            item_type (str): Type of item either: table or udf.
+            name (str): Name of the table or UDF to be created.
+            model_type (str): Type of model to be created could be 'classification', 'object-detection', etc.
+            model (str): Path of the model weigths.
+            **kwargs: Additional keyword arguments for configuring the load operation.
```
Comment (gaurav274) [inline eva/interfaces/relational/db.py:227]: Add support for `if_not_exists.` It is super important. Default set it to `True`

File: eva/interfaces/relational/db.py [hunk 7]
```diff
@@ -208,3 +272,8 @@ async def get_connection(host: str, port: int) -> EVAConnection:
 def connect_remote(host: str, port: int) -> EVAConnection:
     connection = asyncio.run(get_connection(host, port))
     return connection
+
+
+def connect_without_server() -> EVAConnection:
```
Comment (gaurav274) [inline eva/interfaces/relational/db.py:277]: We don't need this

File: eva/parser/utils.py [hunk 8]
```diff
@@ -46,6 +50,30 @@ def parse_load(table_name: str, file_regex: str, format: str, **kwargs):
     return stmt
 
 
+def parse_drop(item_name: str, item_type: str, **kwargs):
+    mock_query = f"DROP {item_type} {item_name}"
```
Comment (gaurav274) [inline eva/parser/utils.py:54]: Please fix this based on the comments on APIs

**Comment by gaurav274** [comment]
@aryan-rajoria Can you also work on porting `query` to use `EvaQuery` APIs like `select`, `filter` etc.

**Comment by jarulraj** [comment]
Subsumed in #805

**Pull request closed without merging**